Docs/Editors.md

```diff
@@ -6,9 +6,11 @@ hide_table_of_contents: true
 Running CSharpier on save is recommended. It will speed up your development time.
 
 ### Visual Studio
-Use the [official 2022 extension](https://marketplace.visualstudio.com/items?itemName=csharpier.CSharpier)  
+Use the [official 2022 extension](https://marketplace.visualstudio.com/items?itemName=csharpier.CSharpier)
```

Comment:
Was this an accidental change? There may be a better way for me to have these on separate lines.

Reply:
Sorry, that was a result of my automatic trim trailing whitespace I've scripted into my editor. I can add it back, if you'd like.

Reply:
I pushed a third commit to this PR breaking the lines in the editor documentation without the use of trailing whitespace. You can use a `\` on a line to trigger a line-break.

Reply:
Thanks, `\` seems like a much better option, I never liked the trailing whitespace solution but didn't bother to go looking.